src/plugins/dashboard/public/plugin.tsx

```diff
@@ -90,6 +91,15 @@ export class DashboardEmbeddableContainerPublicPlugin
     uiActions.attachAction(CONTEXT_MENU_TRIGGER, expandPanelAction);
     const startServices = core.getStartServices();
 
+    const [startCore, startPlugins] = GetStartServicesSync(core.getStartServices);
```

Comment:
This is the main line. It returns *synchronously* start-core and start-plugins.

P.S. It should not be capital `G`, I don't know how it happened :smile: